end_to_end/end_to_end_suite_test.go

```diff
@@ -2195,6 +2195,9 @@ LANGUAGE plpgsql NO SQL;`)
 					numSegments := dbconn.MustSelectString(restoreConn, "SELECT numsegments FROM gp_distribution_policy where localoid = 'schemaone.test_table'::regclass::oid")
 					Expect(numSegments).To(Equal(strconv.Itoa(segmentCount)))
 
+					// check there is no pipe errors on segments
+					errSegments := dbconn.MustSelectString(restoreConn, "SELECT exists (SELECT * FROM gp_toolkit.__gp_log_segment_ext WHERE logtime > now() - '1 min'::interval AND logmessage LIKE 'read err msg from pipe%No such file or directory%' ORDER BY logtime DESC)")
```

Comment:
Here I see another flaky test based on time. Is it possible to filter an error message by pid or oid from toc file or anything else?

Reply:
done